challenges.tex

```diff
@@ -0,0 +1,530 @@
+at least $i$ times.
+The placement of the machines on the field is fixed according to
+\reffig{fig:grasping-field}.
+
+\begin{table}[!htb]
+\centering
+ \begin{tabular}{l|l|l}
+  \multirow{2}{*}{Machines}
+  & \multicolumn{2}{c}{Scoring} \\\cline{2-3}
+  & first repetition
+  & each subsequent repetiton  \\\hline\hline
+  1 & +10 & +2 \\
+  2 & +20 & +2 \\
+  3 & +25 & +2 \\
+ \end{tabular}
+ \caption{Grasping Challenge}
+ \label{tab:challenge-grasping}
+\end{table}
+
+\begin{figure}
+	\centering
+	\begin{tikzpicture}[x=-1cm,y=1cm]
+	\fill[gray!50!white] (2,0) rectangle (5,1);
+	\node at (3.5,0) [anchor=south, font=\tiny, inner sep=1pt] {Insertion Zone};
+	\foreach \i in {0,...,5} {
+		\draw (0,\i) -- (5,\i);
+		\draw (\i,0) -- (\i,5);
+	}
+	\foreach \x in {1,...,5} {
+		\foreach \y in {1,...,5} {
+			\node[font=\tiny,anchor=north west, inner sep=1pt] at (\x,\y) {M\_Z\x\y};
+		}
+	}
+		\node at (4.5,1.5) [circle,fill=magenta, draw=black,inner sep=0.2cm] {};
+		\node at (4.5,2.5) [rectangle,fill=green!30!white, draw=black,inner sep=0.1pt,align=center,font=\tiny] {input\\output};
+		\node at (0.5,0.5) [circle,fill=magenta, draw=black,inner sep=0.2cm] {};
+		\node at (0.5,1.5) [rectangle,fill=green!30!white, draw=black,inner sep=0.1pt,align=center,font=\tiny] {input\\output};
+		\node at (0.5,4.5) [circle,fill=magenta, draw=black,inner sep=0.2cm] {};
+		\node at (1.5,4.5) [rectangle,fill=green!30!white, draw=black,inner sep=0.1pt,align=center,font=\tiny,rotate=90] {input\\output};
+
+\end{tikzpicture}
+\caption{Starting configuration for the grasping challenge.
+	BS is placed at $\texttt{M\_Z12}$, CS1 is placed at $\texttt{M\_Z53}$
+and RS1 is placed at $\texttt{M\_Z25}$.}
+	\label{fig:grasping-field}
+\end{figure}
+
+\subsubsection{Product Challenges}\label{sec:challenge-cx}
+This section covers four types of challenges, instead of just a single one.
+Each challenge corresponds to the production of a product with one of the
+available complexities (C0, C1, C2, C3) in the \ac{RCLL} using either
+one or two \ac{RS}.\\
+For complexities C1, C2 and C3 the accumulated cost for mounting the required
+rings must be equal to 1, 2 and 3, respectively.
+\textbf{Task:} Produce all posted orders.\\
+\begin{table}[!htb]
+ \centering
+ \begin{tabular}{l|l|l}
+  Machines & Challenge type & Scoring \\\hline
+  2 & C0 & 30 \\
+  3 & C1 & 50 \\
+  4 & C1 & 50 \\
+  3 & C2 & 70 \\
+  4 & C2 & 70 \\
+  3 & C3 & 100 \\
+  4 & C3 & 100 \\
+ \end{tabular}
+ \caption{CX Challenge}
+ \label{tab:challenge-cx}
+\end{table}
+
+\subsubsection{Exploration + C0 Challenge}\label{sec:challenge-combine-exp-c0}
+A simple production task on a field with unknown machine positions.
+The challenge is to produce a product of complexity C0 without receiving the
+machine positions at the start of the production phase, resembling a unified
+exploration and production phase.\\
+\textbf{Task:} Produce all posted orders.\\
+Beating this challenges yields 50 points.
```

Comment:
Since it is stated above, I guess this here explains `Exploration + C0`,  `Exploration + C1`,  `Exploration + C2`, `Exploration + C3` and it is possible for a team to play all 4 challanges or does a team has to select one out of those?

Reply:
Ok my fault, seems like in Product Challange positions are published by the refbox, and the combination (Exploration + Production) is only available for C0. I would make that also 4 challanges.

Reply:
Yea the main reason for limiting exploration+Production to a single challenge is to not give it so much impact for the end result as its a new concept. Also there was the concern of having too many challenges to compete in (the idea for the normal production with machine positions is indeed that you can play all 4 of them). That depends on the OC as they are responsible to come up with the organization of the timeslots.

I personally like the concept of Exploration+Production and would not be opposed to have more challenges in that area but i am not sure if it would spread the gap between advanced teams and newer ones too much.

We could add those additional complexities for Exploration+Production extra challenges under the Secondary Challenge category that does not yield any points towards the actual challenge and is just for fun.

Reply:
I think that since the exploration is already part of the game for a long time it is not  new concept and hence would like to see it in the Primary Challanges, how do the other people see this? since there is not a big difference between the Production / Production + Exploration challange in my eyes, as long as you are capable of playing the exploraion phase in RCLL.

For me it is also no where stated that a team has to compete in every challange, so I don't see a problem with more challanges that slots. Like if there are 15 challanges I can select which 10 I want to do. If I select one which yields more points the risk that I fail is higher.

Regarding your thought of the gap, I am well aware that the league is hard to join, but limiting the possibilities for better teams will just result in frustration in the long run.

Reply:
But that is something (among other things that will be clarified as part of the initial start of league meeting, since it is not valid for the challenges to come)

Reply:
Since the secondary challenge is scored separately I don't see any issue with you playing on a smaller field with less machines in this context. (I share Tariks concerns of inflating the potential points teams can score with this challenge)

Reply:
i agree with Peter that it makes sense to set an upper limit regardless. big teams with the capacity to run like 20 slots a day have a huge advantage over smaller teams or those with tighter time constraints. Setting some limit might be a good idea. If 10 works we can do that.

Regarding the secondary challenge stuff: fair point so lets roll with it and put more challenges in the main competition. I will update this. When noone else gives feedback noone can complain :P

Regarding changes to the real game: what about scheduling a TC meeting to discuss it with all interested ones to push rule changes on the way? I can organize that and then we go from there.

Reply:
yea i would prefer to shift TC decision discussions to github as it is way easier to manage this.
I need to dig up the request to answer it. For me there is also no problem with allowing that arm ore abolishing the height restriction altogether.